guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-satellite-server.adoc

```diff
@@ -75,10 +75,26 @@ To update the certificates on a currently running Katello installation, run:
 endif::[]
 +
 Note that you must not access or modify `/root/ssl-build`.
-. From the output of the `katello-certs-check` command, depending on your requirements, enter the `{foreman-installer}` command that installs a new {Project} with custom SSL certificates or updates certificates on a currently running {Project}.
+. Run the command to update certificates on your {ProjectServer}:
 +
-If you are unsure which command to run, you can verify that {Project} is installed by checking if the file `/etc/foreman-installer/scenarios.d/.installed` exists.
```

Comment:
At this point in the process, we expect users to have an existing server installation. So I don't think we need the reference for how to check whether a server is installed.